isb_web/auth.py

```diff
@@ -82,10 +83,30 @@ async def __call__(
             scope["oauth2-claims"] = claims
             scope["oauth2-provider"] = provider
             scope["oauth2-jwt"] = token
-        except starlette_oauth2_api.InvalidToken as e:
-            return await self._prepare_error_response(
-                e.errors, 401, scope, receive, send
-            )
+        except starlette_oauth2_api.InvalidToken:
```

Comment:
The one thing I am not crazy about with this is that we don't know beforehand if it's a JWT or an access token.  So the flow is "check JWT, then if it fails check access token".  If you've got a better way of trying this, @datadavev I'd love to hear it.